packages/runtime/test/dataViews/RelationshipDVO.test.ts

```diff
@@ -101,30 +84,13 @@ describe("RelationshipDVO", () => {
         expect(dvo.name).toBe(dto.peer.substring(3, 9));
         expect(dvo.description).toBe("i18n://dvo.relationship.Active");
         expect(dvo.type).toBe("IdentityDVO");
-        expect(dvo.date).toBe(dto.changes[0].request.createdAt);
+
         expect(dvo.isSelf).toBe(false);
         expect(dvo.relationship!.id).toBe(dto.id);
         expect(dvo.relationship!.direction).toBe("Outgoing");
         expect(dvo.relationship!.status).toBe("Active");
         expect(dvo.relationship!.statusText).toBe("i18n://dvo.relationship.Active");
-        expect(dvo.relationship!.changeCount).toBe(1);
-        const change = dvo.relationship!.changes[0];
-        expect(change.id).toBe(dto.changes[0].id);
-        expect(change.type).toBe("RelationshipChangeDVO");
-        expect(change.status).toBe(dto.changes[0].status);
-        expect(change.statusText).toBe("i18n://dvo.relationshipChange.Accepted");
-        expect(change.request.type).toBe("RelationshipChangeRequestDVO");
-        expect(change.request.createdAt).toBe(dto.changes[0].request.createdAt);
-        expect(change.request.createdBy).toBe(dto.changes[0].request.createdBy);
-        expect(change.request.createdByDevice).toBe(dto.changes[0].request.createdByDevice);
-        expect(change.request.content).toBe(dto.changes[0].request.content);
-        expect(change.isOwn).toBe(true);
-        expect(change.response!.type).toBe("RelationshipChangeResponseDVO");
-        expect(change.response!.createdAt).toBe(dto.changes[0].response!.createdAt);
-        expect(change.response!.createdBy).toBe(dto.changes[0].response!.createdBy);
-        expect(change.response!.createdByDevice).toBe(dto.changes[0].response!.createdByDevice);
-        expect(change.response!.content).toBe(dto.changes[0].response!.content);
-        expect(change.date).toBe(dto.changes[0].response!.createdAt);
+
         expect(dvo.relationship!.templateId).toBe(dto.template.id);
```

Comment:
As written above, I guess we need to expand the AuditLogEntries to AuditLogEntryDVOs and thus need to test it here also

Reply:
I'd postpone this to 
1. another PR
2. when we need it